deps/rabbit/src/rabbit_db.erl

```diff
@@ -91,23 +91,29 @@ pre_init(IsVirgin) ->
     OtherMembers = rabbit_nodes:nodes_excl_me(Members),
     rabbit_db_cluster:ensure_feature_flags_are_in_sync(OtherMembers, IsVirgin).
 
-init_using_mnesia() ->
+init_using_mnesia(_IsVirgin) ->
     ?LOG_DEBUG(
       "DB: initialize Mnesia",
       #{domain => ?RMQLOG_DOMAIN_DB}),
     ok = rabbit_mnesia:init(),
     ?assertEqual(rabbit:data_dir(), mnesia_dir()),
     rabbit_sup:start_child(mnesia_sync).
 
-init_using_khepri() ->
+init_using_khepri(IsVirgin) ->
     case rabbit_khepri:members() of
         [] ->
             timer:sleep(1000),
-            init_using_khepri();
+            init_using_khepri(IsVirgin);
         Members ->
             ?LOG_WARNING(
                "Found the following metadata store members: ~p", [Members],
-               #{domain => ?RMQLOG_DOMAIN_DB})
+               #{domain => ?RMQLOG_DOMAIN_DB}),
+            case IsVirgin of
+                true ->
+                    rabbit_khepri:register_projections();
+                false ->
+                    ok
+            end
     end.
```

Comment:
This starts to be a lot of Khepri-specific knowledge outside of `rabbit_khepri`. Would you mind moving this function to `rabbit_khepri`? E.g. `rabbit_khepri:init/1` like Mnesia.

While here, the log message level should be decreased to NOTICE or INFO probably. Perhaps we can add the equivalent of the "DB: initialize Mnesia" log message too.